doc/install/ides.rst

```diff
@@ -0,0 +1,55 @@
+.. _ide_setup:
+
+IDE integration (VSCode, Spyder, etc.)
```

Comment:
This file is 100% cut-paste other than updating this title from `Python IDE integration` to `
`IDE integration (VSCode, Spyder, etc.)` to be more discoverable (removed Python to make it more compact given the new discoverable term additions)